packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -600,6 +600,45 @@ export class LocalCorpus<
   }
 }
 
+/** A fully mananged {@link Corpus} served by Fixie. */
+export class FixieCorpus<ChunkMetadata extends Jsonifiable = Jsonifiable> implements Corpus<ChunkMetadata> {
+  private static readonly FIXIE_API_URL = 'https://app.fixie.ai/api';
+
+  constructor(private readonly corpus_id: string, private readonly fixie_api_key?: string) {
+    if (!fixie_api_key) {
+      this.fixie_api_key = getEnvVar('FIXIE_API_KEY');
+      if (!this.fixie_api_key) {
+        throw new Error(
+          'You must provide a Fixie API key to access Fixie corpora. Find yours at https://app.fixie.ai/profile'
+        );
+      }
+    }
+  }
+
+  async search(query: string, params?: { limit?: number }): Promise<ScoredChunk<ChunkMetadata>[]> {
+    const response = await fetch(`${FixieCorpus.FIXIE_API_URL}/corpora/${this.corpus_id}:query`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${this.fixie_api_key}`,
+      },
+      body: JSON.stringify({ query_string: query, chunk_limit: params?.limit }),
+    });
+    if (response.status !== 200) {
+      throw new Error(`Fixie API returned status ${response.status}: ${await response.text()}`);
+    }
+    const api_results = await response.json();
```

Comment:
```suggestion
    const apiResults = await response.json();
```

We use camelcase in JS. :smile:

Reply:
Whoops, thanks. It'd be awesome if our formatter or linter could catch that!